### DISCUSSION TITLE
Should all interpolated method names omit Interpolated and non-interpolated methods include raw?

### DISCUSSION THREAD

**Discussion by TwentyFourMinutes** [TwentyFourMinutes/Venflow#10] [Ideas]
Currently we have the following methods: `QueryInterpolatedSingle`, `QueryInterpolatedBatch` and `ExecuteInterpolatedAsync`, their purpose is to avoid SQL injection and to allow for easier parameterized SQL. However it is quite easy to mistakenly call their non-interpolated counter parts.

## Proposal
Omit `Interpolated` in all of the above mentioned method names and instead add `Raw` to their counter part methods e.g. `QueryRawSingle`, `QueryRawBatch` and `ExecuteRawAsync`.

## Pros
- It is a lot less likely to mistakenly to execute queries which are vulnerable to SQL injection.
- It is a lot shorter and easier to read.

## Cons
- It is a breaking change.

I would be happy about any thoughts or suggestions as well as a vote for either 👍 or 👎!

**Comment by Neoz-JourneyLab** [comment]
Raw is quite clear about what is does indeed !
